
From: Jesper Juhl <juhl-lkml@dif.dk>

A tiny patch to make PERFCTR_VIRTUAL default to Y to match the
recommendation given in the help text.  The help has a very clear "Say Y"
recommendation and it doesn't make much sense to not enable this currently
if PERFCTR is set, so it should default to Y, not N as it does currently.

Signed-off-by: Jesper Juhl <juhl-lkml@dif.dk>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/perfctr/Kconfig |    1 +
 1 files changed, 1 insertion(+)

diff -puN drivers/perfctr/Kconfig~make-perfctr_virtual-default-in-kconfig-match-recommendation drivers/perfctr/Kconfig
--- 25/drivers/perfctr/Kconfig~make-perfctr_virtual-default-in-kconfig-match-recommendation	2004-08-25 19:07:17.915862288 -0700
+++ 25-akpm/drivers/perfctr/Kconfig	2004-08-25 19:07:17.919861680 -0700
@@ -34,6 +34,7 @@ config PERFCTR_INIT_TESTS
 config PERFCTR_VIRTUAL
 	bool "Virtual performance counters support"
 	depends on PERFCTR
+	default y
 	help
 	  The processor's performance-monitoring counters are special-purpose
 	  global registers. This option adds support for virtual per-process
_
